app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :author, class_name: 'User'
+
+  def five_most_recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
+  def update_post_counter
+    author.increament!(:PostsCounter)
+  end
```

Comment:
- [ ] Kindly make use of the model callback functions such as `after_save` so that the counters are triggered every time a new post is saved.

```ruby
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```


![counters](https://github.com/derrick1451/Blog-app/assets/53124548/1dd768cb-0655-4e4f-9015-6f1f77aa42a0)